README.md

```diff
@@ -61,29 +63,33 @@ The compatibility matrix for client-go and Kubernetes cluster can be found
 All additional compatibility is only best effort, or happens to still/already be supported.
 
 #### Compatibility matrix
+
 At most, 5 kube-state-metrics and 5 [kubernetes releases](https://github.com/kubernetes/kubernetes/releases) will be recorded below.
 
 | kube-state-metrics | **Kubernetes 1.15** | **Kubernetes 1.16** |  **Kubernetes 1.17** |  **Kubernetes 1.18** |  **Kubernetes 1.19** |
 |--------------------|---------------------|---------------------|----------------------|----------------------|----------------------|
 | **v1.8.0**         |         ✓           |         -           |          -           |          -           |          -           |
 | **v1.9.7**         |         -           |         ✓           |          -           |          -           |          -           |
-| **v2.0.0-alpha.1** |         -           |         -           |          ✓           |          ✓           |          ✓           |
-| **master**         |         -           |         -           |          ✓           |          ✓           |          ✓           |
+| **v2.0.0-alpha.2** |         -           |         -           |          -/✓         |         -/✓          |          ✓           |
+| **master**         |         -           |         -           |          -/✓         |          -/✓         |          ✓           |
 - `✓` Fully supported version range.
 - `-` The Kubernetes cluster has features the client-go library can't use (additional API objects, deprecated APIs, etc).
 
+**Note:** The `v2.0.0-alpha.2+` and `master` releases of kube-state-metrics work on Kubernetes v1.17 and v1.18 excluding Ingress or CertificateSigningRequest resource metrics. If you require those metrics and are on an older Kubernetes version, use v2.0.0-alpha.1 or v1.9.7 kube-state-metrics release. 
+
 #### Resource group version compatibility
+
 Resources in Kubernetes can evolve, i.e., the group version for a resource may change from alpha to beta and finally GA
 in different Kubernetes versions. For now, kube-state-metrics will only use the oldest API available in the latest
 release.
 
 #### Container Image
 
 The latest container image can be found at:
-* `quay.io/coreos/kube-state-metrics:v2.0.0-alpha.1`
-* `k8s.gcr.io/kube-state-metrics/kube-state-metrics:2.0.0-alpha.1`
-* `k8s.gcr.io/kube-state-metrics/kube-state-metrics-arm:2.0.0-alpha.1`
-* `k8s.gcr.io/kube-state-metrics/kube-state-metrics-arm64:2.0.0-alpha.1`
+* `quay.io/coreos/kube-state-metrics:v2.0.0-alpha.2`
+* `k8s.gcr.io/kube-state-metrics/kube-state-metrics:2.0.0-alpha.2`
```

Comment:
@lilic are the gcr.io images going to have a v-prefix in the version similar to the quay.io ones with this release?